src/main/java/com/microsoft/aad/msal4j/HttpHelper.java

```diff
@@ -6,32 +6,63 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.List;
-import java.util.Map;
+import java.util.*;
+
+import static com.microsoft.aad.msal4j.Constants.POINT_DELIMITER;
 
 class HttpHelper {
 
     private final static Logger log = LoggerFactory.getLogger(HttpHelper.class);
+    public static final String RETRY_AFTER_HEADER = "Retry-After";
+    public static final int RETRY_NUM = 2;
+    public static final int RETRY_DELAY_MS = 1000;
+
+    private static String getRequestThumbprint(RequestContext requestContext) {
+        StringBuilder sb = new StringBuilder();
```

Comment:
[As per spec](https://identitydivision.visualstudio.com/DevEx/_git/AuthLibrariesApiReview?path=%2FService%20protection%2FIntial%20set%20of%20protection%20measures.md&version=GBdev&_a=preview&anchor=terminology), this should include the authority as well.

Reply:
it does include authority